pychromecast/socket_client.py

```diff
@@ -79,10 +79,13 @@ class InterruptLoop(Exception):
     """ The chromecast has been manually stopped. """
 
 
-def _json_from_message(message):
-    """ Parses a PB2 message into JSON format. """
+def _dict_from_message(message):
+    """ Parses a PB2 message as a JSON dict. """
     try:
-        return json.loads(message.payload_utf8)
+        dict_data = json.loads(message.payload_utf8)
+        if not isinstance(dict_data, dict):
+            return {}
```

Comment:
Should we log a warning or at least debug?

Reply:
Fixed!